microsoft-edge/extensions-chromium/developer-guide/declare-permissions.md

```diff
@@ -89,6 +91,7 @@ The following table lists the currently available permission strings to use in y
 | `scripting` | Gives your extension access to the `chrome.scripting` API. |
 | `search` | Gives your extension access to the `chrome.search` API. |
 | `sessions` | Gives your extension access to the `chrome.sessions` API. |
+| `sidePanel` | Gives your extension access to the `chrome.sidePanel` API. |
```

Comment:
in file declare-permissions.md
@Bhuvana-Priya - review this row i added

Reply:
entered fresh [Comment](https://github.com/MicrosoftDocs/edge-developer/pull/2648/files#r1234516760)